v1/pkg/client/api/openapi.yaml

```diff
@@ -2695,19 +2695,23 @@ components:
     AvailableResources:
       example:
         Networks:
-        - IPPoolID: IPPoolID
+        - VNI: 6
```

Comment:
Not a big deal, but VNIs are typically large numbers. Is there a way to specify example values? VNI: 15006 would be more common.